Invalid example code in jump statements, return ref

### Type of issue

Code doesn't work

### Description

The ref returns section contains the following example code: `int[] xs = [10, 20, 30, 40];`

Output: `(1,12): error CS1525: Invalid expression term '['`

It should be `int[] xs = {10, 20, 30, 40};`


### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/jump-statements#ref-returns

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/statements/jump-statements.md

### Document Version Independent Id

e2587107-1293-4aa1-8f74-6417367d76e7

### Article author

@BillWagner

### Metadata

* ID: 5c1793eb-a1cc-62d8-2009-2d69eefa9413 
* Service: **dotnet-csharp**
* Sub-service: **lang-reference**

--- a/docs/csharp/language-reference/statements/snippets/jump-statements/ReturnStatement.cs
+++ b/docs/csharp/language-reference/statements/snippets/jump-statements/ReturnStatement.cs
@@ -52,7 +52,7 @@ public static class ReturnStatement
     private static void RefReturn()
     {
         // <RefReturn>
-        int[] xs = [10, 20, 30, 40];
+        int[] xs = new int [] {10, 20, 30, 40 };
         ref int found = ref FindFirst(xs, s => s == 30);
         found = 0;
         Console.WriteLine(string.Join(" ", xs));  // output: 10 20 0 40

